doc/getting_started.md

```diff
@@ -0,0 +1,556 @@
+# Jörmungandr Wallet and Chain interactions
+
+- list of jörmungandrwallet lib functions
+- what and how are used for/to interact with the external network.
+
+## Wallet used in examples
+
+```json
+"wallet": {
+  "Note":      "DAEDALUS",
+  "Mnemonics": "tired owner misery large dream glad upset welcome shuffle eagle pulp time",
+  "Funds":
+  [
+    {
+      "Address": "DdzFFzCqrhsktawSMCWJJy3Dpp9BCjYPVecgsMb5U2G7d1ErUUmwSZvfSY3Yjn5njNadfwvebpVNS5cD4acEKSQih2sR76wx2kF4oLXT",
+      "Value":   10000,
+    },
+    {
+      "Address": "DdzFFzCqrhsg7eQHEfFE7cH7bKzyyUEKSoSiTmQtxAGnAeCW3pC2LXyxaT8T5sWH4zUjfjffik6p9VdXvRfwJgipU3tgzXhKkMDLt1hR",
+      "Value":   10100,
+    },
+    {
+      "Address": "DdzFFzCqrhsw7G6njwb8FTBxVCh9GtB7RFvvz7KPNkHxeHtDwAPT2Y6QLDLxVCu7NNUQmwpAfgG5ZeGQkoWjrkbHPUeU9wzG3YFpohse",
+      "Value":   1, // this will be ignored upon conversion
+    },
+  ]
+}
+```
+
+### The fees in the sample chain
+
+```json
+  "fees": {
+    "certificate": 5,
+    "coefficient": 3,
+    "constant": 2
+  },
+```
+
+---
+
+## Main core api
+
+- [wallet_recover](#wallet_recover)
+- [wallet_id](#wallet_id)
+- [wallet_retrieve_funds](#wallet_retrieve_funds)
+- [wallet_total_value](#wallet_total_value)
+- [wallet_convert](#wallet_convert)
+- [wallet_convert_ignored](#wallet_convert_ignored)
+- [wallet_convert_transactions_size](#wallet_convert_transactions_size)
+- [wallet_convert_transactions_get](#wallet_convert_transactions_get)
+- [wallet_set_state](#wallet_set_state)
+
+---
+
+### wallet_recover
+
+- **input**:
+  - `mnemonics` - utf8 string (already normalized **NFKD**) in **english**
+  - `password` - optional
+- **output**:
+  - `wallet` - recovered from mnemonics
+  - `error` - returned in case the mnemonics are not valid (invalid length or checksum)
+
+> **NFKD** - **N**ormalization **F**orm: Compatibility (**K**) **D**ecomposition.
+> Characters are decomposed by compatibility, and multiple combining characters are arranged *in a specific order*.
+
+This function is used to retrieve a wallet from the given mnemonics and password.
+It will work for all yoroi, daedalus and other wallets as it will try every kind of wallet anyway
+
+Can also be used to recover a wallet even after you have transferred all the funds to the new format (see the [wallet_convert](#wallet_convert) function), just keep in mind that it will reset the wallet internal state.
+
+---
+
+### wallet_id
+
+- **input**:
+  - `wallet` - retireved from mnemonics ([wallet_recover](#wallet_recover))
+- **output**:
+  - `walled_id` - (aka accountID) used against the blockchain/explorer to retrieve the state of the wallet
+  - `error` - returned in case the wallet is not yet recovered
```

Comment:
I think you don't technically have a wallet to ask for the id until you recover it. Although you may be talking about something different than what I have in mind.

Reply:
We may not have a wallet on chain yet  until we send the transactions for the convertion, but afaik the wallet_id is immediately available/derived from wallet_recover.

I tried to order the functions based on the immediate availability.